test/org/sagebionetworks/bridge/services/AuthenticationServiceTest.java

```diff
@@ -132,6 +141,43 @@ public void signInInvalidCredentials() throws Exception {
         authService.signIn(study, TEST_CONTEXT,
                 new SignIn.Builder().withStudy(study.getIdentifier()).withEmail("foobar").withPassword("bar").build());
     }
+    
+    @Test
+    public void signUpWithIntentToParticipate() throws Exception {
+        String email = TestUtils.makeRandomTestEmail(AuthenticationServiceTest.class);
+        StudyParticipant participant = new StudyParticipant.Builder()
+                .withEmail(email)
+                .withPassword("P@ssword`1").build();
+        IdentifierHolder holder = null;
```

Comment:
We want to move away from real dependencies in unit tests. Would it be better to have a test in AuthenticationServiceMockTest that tests dataflow and wiring, and have this test be in Integ Tests instead?